README.md

```diff
@@ -1,4 +1,14 @@
 # hw01
 
-## Brief Biography
-My name is Brenda Castaneda. I am a third-year in the college and I am majoring in *Public Policy* and *Latin American and Carribean Studies*. 
\ No newline at end of file
+### About Me
+My name is Brenda Castaneda. I am a third-year in the college and I am majoring in *Public Policy* and *Latin American and Caribbean Studies*. Some of my policy interest areas are education, immigration, and foreign policy. 
+
+### Interest in this course
+I am taking this course to satisfy the Methods requirement for the Public policy major. I took [Social Science Inquiry](http://collegecatalog.uchicago.edu/search/?P=SOSC%2013100-13200-13300) last year and was introduced to social science research tools and data analysis using R. Computing for the social sciences was recommended to me by a friend and I am looking forward to brushing up on and expanding upon what I learned in SSI last year.  
+
+### Quaratine Activities
+I am currently in Chicago but spent my summer in quarantine at home in Dallas. Here's a list of some of the things I did to make life a little less boring:
+
+* taking long walks by the lake shore
+* baking
+* painting/DIY projects 
```

Comment:
Overall, nice job! It could have been nice to have a picture, but obviously that wasn't required!